nbextensions/editorCell/styles/main.css

```diff
@@ -1,419 +0,0 @@
-/* Wrapper class
```

Comment:
Moved all these styles into `appCell.css`. The majority mimic styles that were already in that file.